include/gantry/core/can_task.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "can/core/freertos_can_dispatch.hpp"
+#include "can/core/freertos_sender_task.hpp"
+#include "can/core/message_handlers/device_info.hpp"
+#include "can/core/message_handlers/motor.hpp"
+#include "can/core/message_handlers/move_group.hpp"
+#include "can/core/message_handlers/move_group_executor.hpp"
+#include "common/core/freertos_message_queue.hpp"
+#include "common/core/freertos_task.hpp"
+#include "motor-control/core/motor.hpp"
+
+namespace can_bus {
```

Comment:
once we refactor pipettes and head to have interfaces & the tasks living in core, then we should be able to have this header file shared, right?

Reply:
Possibly.

Reply:
I hope so, but I get the feeling that the interfaces will evolve.